Shared/AppInfo.swift

```diff
@@ -62,4 +62,8 @@ class AppInfo {
     static var isBetaBuild: Bool {
         return (Bundle.main.infoDictionary?["CFBundleIdentifier"] as? String)?.contains("enterprise") ?? false
     }
+    
+    open class func testRequestsReset() -> Bool {
+        return ProcessInfo.processInfo.arguments.contains("RESET_PREFS")
```

Comment:
@oliviabrown9 question - why did you have to add "RESET_PREFS"? Why couldn't you check for "testmode" here?

Reply:
@Sdaswani I did it so that we would have flexibility with other tests not to reset preferences (since it increases test time). For example, we might not need to reset the preferences for snapshot tests. I can change it to always just be based on testmode, but that would change it for all test types (which we may or may not want to do). What are your thoughts?

Reply:
That makes sense!